server/utils/index.js

```diff
@@ -1,5 +1,12 @@
 'use strict';
 
+const COMBINED_UID_JOINSTR = ', ';
```

Comment:
So this will be used in the filename I think we should change it's value.
It should be something that is not used in the value of the UID, otherwise you can't correctly split them.

Maby `+`. What do you think?

Reply:
The joined UID will be presented in the UI. Therefore I have chosen `', '` as the join string so it looks nice in the UI.

I think any character could potentially be used in the name so it should really be something very unique but at the same time this would make it more 'uggly' in the UI unless we replace it there again.

Another Idea would be to escape the usage of the join string in the values but then it is hard to restore the original value for the where filter. May I check if we always can restore the unescaped versions from the stored json file.

Reply:
Seems like I found another working solution. I agree with replacing it in the frontend makes sense! Didn't got the frontend dev part working now so I may leave this for another PR?